CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 /k-distribution/include/kframework/builtin/* @dwightguth @ehildenb
-/Jenkinsfile @runtimeverification/admin
-/.github/workflows/* @runtimeverification/admin
+/Jenkinsfile @runtimeverification/admin @F-WRunTime
```

Comment:
I think it's probably more appropriate to just add Freeman to the admin team than to explicitly hardcode his username here.